test/Models/resnet50v1.mlir

```diff
@@ -1,5 +1,6 @@
 // RUN: tpp-opt %s -default-tpp-passes | \
 // RUN: FileCheck %s
+// CHECK: XFAIL:*
```

Comment:
And this one. They should not fail. If the output is different, then we need to understand what it is and fix the checks.